libdnf/dnf-context.cpp

```diff
@@ -890,6 +911,23 @@ dnf_context_set_repo_dir(DnfContext *context, const gchar *repo_dir)
     priv->repo_dir = g_strdup(repo_dir);
 }
 
+/**
+ * dnf_context_set_vars_dir:
+ * @context: a #DnfContext instance.
+ * @vars_dir: the vars directories, e.g. ["/etc/dnf/vars"]
+ *
+ * Sets the repo variables directory.
+ *
+ * Since: 0.26.1
+ **/
+void
+dnf_context_set_vars_dir(DnfContext *context, gchar **vars_dir)
+{
+    DnfContextPrivate *priv = GET_PRIVATE(context);
+    g_strfreev(priv->vars_dir);
+    priv->vars_dir = g_strdupv(vars_dir);
```

Comment:
After change of `vars_dir` to `const gchar * const *` this will be needed:
`priv->vars_dir = g_strdupv(const_cast<gchar **>(vars_dir));`

And adjust in the header file too.